src/keycloak/chart/templates/statefulset.yaml

```diff
@@ -90,11 +95,11 @@ spec:
               value: edge
             
             # X509 configuration
-            # - name: KC_SPI_TRUSTSTORE_FILE_FILE
-            #   value: /truststore/truststore.jks
-            # - name: KC_SPI_TRUSTSTORE_FILE_PASSWORD
-            #   value: changeit
-            
+            - name: KC_SPI_TRUSTSTORE_FILE_FILE
+              value: /opt/keycloak/conf/truststore.jks
+            - name: KC_SPI_TRUSTSTORE_FILE_PASSWORD
```

Comment:
since we're [using the default](https://www.keycloak.org/server/enabletls#_setting_the_truststore_password), think we can drop this

Reply:
I added this as a helm value in case someone is loading an alternate truststore for their environment and "secured" it with a real password. This way they can override the helm value with their bundle to use whatever password is needed. Thoughts on that?

I could wrap this in a conditional and leave it off in the default case to keep things a bit cleaner though.

Reply:
Apparently despite the default in docs this is required, or something else is nulling out the default:
```
keycloak ERROR: Unexpected error when starting the server in (development) mode
keycloak ERROR: Failed to start quarkus
keycloak ERROR: Attribute 'password' missing in 'truststore':'file' configuration
```

Will look at that a bit more and open upstream issue if that's a bug/bad docs.